### PR TITLE
Uptake oryx startup script generator

### PR DESCRIPTION
**Pull request by gpcastro**

### CODE REVIEW

**Comment by gpcastro** [comment]
@kichalla Can you please take look as well?

File: debian-8/init_container.sh [hunk 1]
```diff
@@ -19,51 +19,12 @@ cat /etc/motd
...
-        fi
-    fi
-fi
-
-echo Could not find .csproj or .dll. Using default.
-cd /defaulthome/hostingstart
-exec dotnet hostingstart.dll
+appPath="/home/site/wwwroot/"
+sourcePath="/home/site/repository/"
+startupCommantPath="/opt/startup/startup.sh"
```
Comment (kichalla) [inline debian-8/init_container.sh]: typo: startupCommantPath => startupCommandPath

Comment (kichalla) [reply debian-8/init_container.sh]: similarly in other file too

File: debian-8/init_container.sh [hunk 2]
```diff
@@ -19,51 +19,12 @@ cat /etc/motd
...
+defaultAppPath="/defaulthome/hostingstart/hostingstart.dll"
+oryxArgs="-appPath $appPath -sourcePath $sourcePath -output $startupCommantPath -defaultAppFilePath $defaultAppPath -bindPort $PORT -userStartupCommand '$@'"
```
Comment (kichalla) [inline debian-8/init_container.sh]: nit: long line

Comment (kichalla) [inline debian-8/init_container.sh]: Should we have `runFromPath`? (or maybe in another PR I think)

**Review by kichalla — APPROVED** [review]

**Comment by kichalla** [comment]
cc @JennyLawrance as in our last meeting that she mentioned about trying out Oryx with 2.2 version..but here we would be changing for 2.1 version too.

File: debian-8/init_container.sh [hunk 3]
```diff
@@ -19,51 +19,17 @@ cat /etc/motd
...
-    fi
+appPath="/home/site/wwwroot/"
+sourcePath="/home/site/repository/"
+startupCommandPath="/opt/startup/startup.sh"
+defaultAppPath="/defaulthome/hostingstart/hostingstart.dll"
+oryxArgs="-appPath $appPath -sourcePath $sourcePath -output $startupCommandPath -defaultAppFilePath $defaultAppPath \
+    -bindPort $PORT -userStartupCommand '$@'"
+
+if [ "$APP_SVC_RUN_FROM_COPY" = true ]; then
+    oryxArgs="-runFromPath /tmp $oryxArgs"
```
Comment (kichalla) [inline debian-8/init_container.sh]: Probably having a subdirectory is better here

File: debian-8/init_container.sh [hunk 4]
```diff
@@ -19,51 +19,17 @@ cat /etc/motd
...
-                echo Found: $DLL
-                exec dotnet "$DLL"
-            fi
-        fi
-    fi
+appPath="/home/site/wwwroot/"
+sourcePath="/home/site/repository/"
+startupCommandPath="/opt/startup/startup.sh"
+defaultAppPath="/defaulthome/hostingstart/hostingstart.dll"
+oryxArgs="-appPath $appPath -sourcePath $sourcePath -output $startupCommandPath -defaultAppFilePath $defaultAppPath \
```
Comment (kichalla) [inline debian-8/init_container.sh:26]: With my check-in yesterday into Oryx, `sourcePath` is no longer required or available as switch...probably this needs to go into a different branch?

Comment (gpcastro) [reply debian-8/init_container.sh:26]: yes I know, but that is not in prod yet.

**Review by JennyLawrance — APPROVED** [review]

**Pull request merged**